saleor/graphql/core/types/common.py

```diff
@@ -301,3 +302,21 @@ def resolve_type(cls, instance, _info):
             # <DjangoModel>: <GrapheneType>
         }
         return MODEL_TO_TYPE_MAP.get(type(instance))
+
+
+class InvoiceJobInterface(Job):
+    status = InvoiceStatusEnum(description="Job status.", required=True)
```

Comment:
We have two options here
1) leave it modified as is (new interface)
2) Remove `status` from here and just use common Job interface `status` but that doesn't have entity like `pending_delete` so we would have to add some filed like "pending_target" on the job. Or an option "pending_delete" to job. Moving to Job's `status` benefits in that we have same `status` for new interface. What you guys think is better?

Reply:
I would prefer to have a single common interface. We can extend the base `Job` if needed (as long as the statuses are generic).